pkgs/development/python-modules/nose-focus/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, six, nose, nose-of-yeti
+, nose-pattern-exclude, which }:
+
+buildPythonPackage rec {
+  pname = "nose_focus";
```

Comment:
```suggestion
  pname = "nose-focus";
```

Reply:
Thanks, fixed now! I just used the "official" PyPI name here as I thought it should be used here, although I used the normalised name elsewhere.